ddr/ddr/Models/GameEntities/Rider.swift

```diff
@@ -91,7 +91,7 @@ class Rider: SKSpriteNode {
         }
         
         // FIXME: rotation only works for 1 orientation (home button left)
```

Comment:
if you change how the code is working it would also be good to change the comment